README.md

```diff
@@ -20,10 +20,20 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   cd ..
   catkin_make
   ```
+  
+### Application can work at two modes:
+-  Local mode
+  -  All the ROS nodes run at the same machine.
+-  Network mode
+  -  ROS nodes run at two different machine: Server and Client.
+  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
+  -  Server runs all other ROS nodes except two nodes running on client.
```

Comment:
Thank you for clarifying this.

I'm clear now, when I was saying rviz as simulated robot, actually that is   industrial_robot_simulator, and **simulated robot** is the **translucent model** showing in Rviz(UI and robot visualization).

Actually In the current code, I simulated multiple **simulated robots nodes** by launching different "industrial_robot_simulator" using different **namespaces**, then RVIZ in different namespace will launch corresponding **simulated robots nodes**.

But I didn't change anything on real robot part, Now I just begin to use real robot Youbot to understand how to control real robots.

> <!-- Bring up simulated robot that can be visualized under the tf namespace
>        Allows for previewing paths generated before execution. -->
>   <include file="$(find industrial_robot_simulator_service)/launch/start_simulation_server.launch" > <arg name="namespace" **value="$(arg namespace)"**/>
>   </include>
> 
>